src/styles/utils/_vars.scss

```diff
@@ -1 +1,12 @@
 $c-gray: #eee;
+$custom-green: #1a5a4c;
+$custom-gray: #687480;
+$custom-orange: #f14a27;
+$custom-darkgray: #4e4e4e;
+$custom-dark: #0f0e08;
+$custom-padding: 20px;
+$custom-grey: #e0e0e0;
```

Comment:
There are two variables representing different shades of gray: $custom-gray and $custom-grey. To avoid confusion and improve readability, consider renaming them to something more descriptive, such as $custom-light-gray and $custom-dark-gray.